Source/Plugins/Core/com.equella.core/scalasrc/com/tle/web/api/search/model/SearchResultItem.scala

```diff
@@ -89,3 +91,11 @@ case class SearchResultAttachment(
     links: java.util.Map[String, String],
     filePath: Option[String]
 )
+
+/**
+  * Model class providing DRM related status.
+  *
+  * @param termAccepted Whether terms have been accepted or not.
+  * @param isAuthorised Whether user is authorised to access Item or accept DRM.
+  */
+case class DrmStatus(termAccepted: Boolean, isAuthorised: Boolean)
```

Comment:
I've noticed a few places above the singular `term` has been used rather than `terms`. I think it should be the plural. Maybe start here with:

```suggestion
case class DrmStatus(termsAccepted: Boolean, isAuthorised: Boolean)
```

And then address all other places.